examples/text_labels_and_annotations/mathtext_fontfamily_example.py

```diff
@@ -0,0 +1,38 @@
+"""
+========================
+Math Fontfamily Example
+========================
+
+A simple example showcasing the new *math_fontfamily* parameter that can
+be used to change the family of fonts for each individual text
+element in a plot. If no parameter is set, the global value
+:rc:`mathtext.fontset` will be used.
+"""
+
+import matplotlib.pyplot as plt
+
+plt.figure(figsize=(9, 4))
+
+# A simple plot for the background.
+plt.plot(range(11), color="0.9", label=r'Math Font Example')
+
+# Here we hold the text, we will mix normal text and math text
+msg = r"This is a normal text and "
+msg+= r"$this\ is\ a\ text\ in\ math\ mode,\ look,\ I\ can\ do\ this\ "
+msg+= r"\int_{0}^{\infty } x^2 dx !$"
+
+# Now we set the text to be shown
+plt.text(0, 6, msg, size=11, math_fontfamily='cm')
+
+# The parameter math_fontfamily migth be aliased to math_font, the
+# code below will work just as well.
+plt.text(0, 3, msg, size=11, math_font='dejavuserif')
+
+# Also, we can call *math_fontfamily* and *math_font* in most places where
+# there is text, like in the title:
+titleMsg = r"$This\ is\ a\ title\ in\ math\ mode,\ look:\ "
+titleMsg+= r"\int_{0}^{\infty } x^2 dx !$"
+
+plt.title(titleMsg, math_font='stixsans', size=14)
```

Comment:
Keep it short:
```suggestion
plt.title(r"$Title\ in\ math\ mode:\ \int_{0}^{\infty } x^2 dx$"
          math_font='stixsans', size=14)
```

Reply:
I thought a lengthier text would make the fonts easier to distinguish by the reader. Perhaps with the new fonts that wont be necessary anymore.

Reply:
Watching for obvious distinguishability is a good idea and the key criterion!

It's even better if that can be achieved with fewer words/less clutter.

A similar thing holds for the texts: IMHO

`"Normal Text. $Text\ in\ math\ mode:\ \int_{0}^{\infty } x^2 dx$"` should be sufficient. But I leave it up to you deciding for the most distinguishable text.

Reply:
I agree, specially after the fonts were changed the longer text was actually being a nuisance. By shortening it the example will be clearer and I will be able to unstretch the figure (that was a bit ugly).